src/cogs/fun.py

```
@@ -0,0 +1,47 @@
+            default=6,
+            min_value=2,
+            max_value=100,
+        ),
+    ) -> None:
+        """Roll a dice and reply with the result."""
+        await interaction.response.defer()
+        result = self._roll_dice(sides)
+        await interaction.followup.send(f"🎲 You rolled **{result}** (1-{sides})")
+
+
+async def setup(bot: commands.Bot) -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)